app/javascript/components/copy-catalog-form/copy-catalog-form.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import createSchema from './copy-catalog-form.schema';
+import { http } from '../../http_api';
+
+class CopyCatalogForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoaded: false,
+    };
+  }
+
+  componentDidMount() {
+    this.setState(() => ({
+      schema: createSchema(),
+      initialValues: {
+        name: 'Copy of',
+      },
+      isLoaded: true,
+    }));
+  };
+
+  handleError = (error) => {
+    const { data: { error: { message } } } = error;
+    return !!message ? message : __('There was an error in copying. Item is not valid or Ansible.');
```

Comment:
This error message is little bit confusing. I needed to read it several times to understand it correctly.

Reply:
@ZitaNemeckova maybe message should be "Catalog Bundle or Ansible Playbook Items can not be copied."

Reply:
I went with `Selected item can not be copied. Because it's Ansible Playbook or not valid.`